doc/user/install.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Install"
+description: "Get started with Materialize"
+menu: "main"
+weight: 2
+---
+
+You can access Materialize through the `materialized` binary, which you can install on macOS and Linux.
+
+## macOS installation
+
+### Homebrew
+
+Assuming you've installed [Homebrew](https://brew.sh/):
+
+```shell
+brew install MaterializeInc/materialize/materialized
+```
+
+### curl
+
+```shell
+curl -L https://downloads.mtrlz.dev/materialized-latest-x86_64-apple-darwin.tar.gz | tar -xzC /usr/local --strip-components=1
+```
+
+## Linux installation
+
+### apt
+
+```shell
+apt install materialized
```

Comment:
There's a missing configuration bit here to add the custom APT repository. Let me see if I can dig it up.

Reply:
```echo "deb [trusted=yes] https://packages.materialize.io/apt/ /" > \
/etc/apt/sources.list.d/fury.list```

Reply:
https://gemfury.com/help/apt-repository/#using-apt-with-gemfury